www/docs/src/content/docs/how-it-works/index.mdx

```diff
@@ -0,0 +1,79 @@
+---
+title: "How does it work!?"
+description: "How does Astro StudioCMS Work?  Well here is all the answers!"
```

Comment:
Not sure how fond I am of this description. First suggestion fixes the existing one, second suggestion is a random proposal.

```suggestion
description: "How does Astro StudioCMS Work? Well, here are all the answers!"
```

```suggestion
description: "Learn how Astro StudioCMS helps you manage and deliver your content by exploring a comprehensive breakdown of it's fundamental components."
```

Reply:
I really do like the second one...